tests/mtest/src/main/scala/tests/BaseSuite.scala

```diff
@@ -32,18 +32,21 @@ class BaseSuite extends munit.FunSuite with Assertions {
 
   override def munitTimeout: Duration = Duration("10min")
 
-  // NOTE(olafur): always ignore flak test failures.
+  // NOTE(olafur): always ignore flaky test failures.
   override def munitFlakyOK = true
-  override def munitRunTest(
-      options: TestOptions,
-      body: => Any
-  ): Any = {
-    if (Properties.isWin && options.tags(FlakyWindows)) {
-      munitFlaky(options, body)
-    } else {
-      super.munitRunTest(options, body)
-    }
-  }
+
+  case class Rerun(count: Int) extends munit.Tag("Rerun")
```

Comment:
Unused?

Reply:
Ahh yea. Good catch, I forgot to remove it.